lib/active_merchant/billing/gateways/conekta.rb

```diff
@@ -12,7 +12,7 @@ class ConektaGateway < Gateway
 
       def initialize(options = {})
         requires!(options, :key)
-        options[:version] ||= '0.3.0'
+        options[:version] ||= '1.0.0'
```

Comment:
I've already updated the PR as requested! Let me know if you need anything else!